Cargo.toml

```diff
@@ -7,6 +7,8 @@ members = [
     "apps/trivium",
     "concrete-csprng",
     "backends/tfhe-cuda-backend",
+    "versionable",
+    "versionable/derive"
```

Comment:
if the crate is called tfhe-something then let's have a tfhe-versionable dir wdyt ?

or if you want to avoid issues when navigating and it's a "helper" crate then we could have a new dir for helper crates, something like "utils" or something

Reply:
The idea was to make it clear that this is not something cryptography oriented, so yes I think moving it into a "utils" subdir is better !